torch/_dynamo/__init__.py

```diff
@@ -64,6 +64,10 @@ def reset() -> None:
     compilation_metrics.clear()
     reset_frame_count()
 
+    import linecache
+
+    linecache.clearcache()
```

Comment:
hmm?

Reply:
It makes sense to me that we can clear the linecache when we reset dynamo. Are you suggesting that we do not?

Reply:
I mean... it doesn't seem harmful but OTOH is this actually going to do anything?